test/unit/mongo_client_options.test.js

```diff
@@ -450,4 +451,32 @@ describe('MongoOptions', function () {
       );
     });
   });
+
+  context('when loadBalanced=true is in the URI', function () {
```

Comment:
should there be analogous tests for when it's passed through the options object?

Reply:
Actually it's only allowed in the URI, not the options object, so I've added tests for those cases and fixed the behaviour to throw when provided in the options object.

Reply:
Hmm, how is the test in L146 still passing given that we're passing it in in L83?

Reply:
Nice catch. It wasn't passing so I've now updated the tests for the correct expectations.